docs/PumpChecklist.md

```diff
@@ -0,0 +1,213 @@
+    - `[ ]` no insulin
+        - `[ ]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[ ]` low power
+    - `[ ]` no power
+        - `[ ]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[ ]` occlusion
+        - `[ ]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[ ]` no delivery
+        - `[ ]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[ ]` auto-off
+        - `[ ]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[ ]` over limit (i.e., max bolus exceeded through override)
+    - `[ ]` other alarm types (details to be provided in `payload` object)
+  - `[ ]` prime events
+    - `[ ]` prime target = tubing
+    - `[ ]` prime target = cannula 
+    - `[ ]` prime targets not differentiated
+    - `[ ]` prime volume in units of insulin
+  - `[ ]` reservoir change (or reservoir rewind)
+    - `[ ]` needed to infer a suspend (stoppage of all insulin delivery)
+  - `[ ]` status events (i.e., suspend & resume)
+    - `[ ]` suspensions of insulin delivery are represented as (interval) events with a duration OR
+    - `[ ]` suspensions of insulin delivery are represented as pairs of point-in-time events: a suspension and a resumption
+    - `[ ]` reason/agent of suspension (`automatic` or `manual`)
+    - `[ ]` reason/agent of resumption (`automatic` or `manual`)
+  - calibrations: see [the CGM checklist](CGMChecklist.md) instead
+  - `[ ]` time changes (presence of which is also in the [BtUTC section](#bootstrapping-to-utc) below)
+    - `[ ]` device display time `from` (before change) and `to` (result of change)
+    - `[ ]` agent of change (`automatic` or `manual`)
+    - `[ ]` timezone
+    - `[ ]` reason for change (read from device)
+
+Device-specific? (Add any device-specific notes/additions here.)
+
+#### SMBG
+
+  - `[ ]` blood glucose value
+  - `[ ]` subType (`linked` or `manual`)
+  - `[ ]` units of value (read from device, not hard-coded)
+  - `[ ]` out-of-range values (LO or HI)
+  - `[ ]` out-of-range value thresholds (e.g., often 20 for low and 600 for high on BGMs)
+
+No Tidepool data model yet:
+
+  - `[ ]` meal tag (i.e., pre- or post-meal)
+  - `[ ]` other/freeform tags
+  - `[ ]` categorization of value according to BG target(s) from settings
+
+Device-specific? (Add any device-specific notes/additions here.)
+
+#### Settings
+
+  - `[ ]` basal schedules
+    - `[ ]` name of basal schedule OR
+    - `[ ]` name of settings profile
+    - `[ ]` each schedule as a set of objects each with a rate and a start time
+  - `[ ]` name of currently active basal schedule
+  - `[ ]` units of all blood glucose-related fields (read from device, not hard-coded)
+  - `[ ]` units of all carb-related fields (read from device, not hard-coded)
+  - `[ ]` carb ratio(s)
+    - `[ ]` name of settings profile
+    - `[ ]` (one or more) set(s) of objects each with a ratio (amount) and a start time
+  - `[ ]` insulin sensitivity factor(s)
```

Comment:
All of the things here are possibly not implemented by the data protocol/spec. Just leave the checkbox item blank (as per key at top) if not part of a device's data protocol/spec.